bundles/framework/divmanazer/component/SelectList.js

```diff
@@ -139,6 +139,12 @@ Oskari.clazz.define('Oskari.userinterface.component.SelectList', function (id) {
      */
     disableOptions: function (ids) {
         var chosen = this.element.find('select');
+        
+        var state = this.getOptions();
+        for (var i = 0; i < state.disabled.length; i++) {
+            jQuery(state.disabled[i]).attr('disabled', false);
+        }
```

Comment:
This could be a call to reset(). If you don't want to trigger the chosen:updated event you can send use some "suppressEvent" kind of optional boolean param on reset for this.